Assets/Scripts/FloatingOrigin/ThreeDTilesWorldTransformShifter.cs

```diff
@@ -27,35 +38,35 @@ public override void PrepareToShift(WorldTransform worldTransform, Coordinate fr
             var contentComponents = transform.GetComponentsInChildren<Content>();
             foreach (Content contentComponent in contentComponents)
             {
-                foreach (Transform child in contentComponent.transform)
-                {
-                    var baseCoordinate = new Coordinate(child.position);
-                    tilesToShift.Add(
-                        child, 
-                        baseCoordinate
-                    );
-                }
+                Coordinate position = new Coordinate(contentComponent.transform.position);
```

Comment:
Gaat dit goed? Dat we niet meer de children verplaatsen maar de content components? Die staan namelijk op 0,0,0?

Reply:
Na tweede review snap ik nog niet waarom de content containers nu worden verplaatst ipv de tiles zelf; dat is misschien nog wel even goed om te verifieren

Reply:
toelichting toegevoegd om het achterliggende idee toe te lichten